From 1b90e48b21b120d1fb255c982542f796b3d42214 Mon Sep 17 00:00:00 2001
From: James Smith <jpdsmith@google.com>
Date: Tue, 6 Dec 2022 14:26:10 +0000
Subject: [PATCH] Removes unnecessary permission from the EmergencyInfo app.

Test: Manually tested EmergencyInfo with multiple users. Adding
contacts, medical info, editing user name and photo and eSos all still
work.

Bug: 248251018
Bug: 252995513
Change-Id: I1cdef8dd9a1a0432bbafdfe9e98756dd11cfd092
(cherry picked from commit 00dbb36b79cc7e611ac0c3387fa692225299c72b)
Merged-In: I1cdef8dd9a1a0432bbafdfe9e98756dd11cfd092
---
 AndroidManifest.xml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 0e2ae94..5780a17 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -28,7 +28,6 @@
     <uses-permission android:name="android.permission.CALL_PRIVILEGED"/>
     <uses-permission android:name="android.permission.READ_CONTACTS"/>
     <uses-permission android:name="android.permission.MANAGE_USERS"/>
-    <uses-permission android:name="android.permission.INTERACT_ACROSS_USERS_FULL"/>
     <uses-permission android:name="android.permission.WRITE_SECURE_SETTINGS"/>
 
     <uses-sdk android:minSdkVersion="21"/>
-- 
2.39.0.rc1.256.g54fd8350bd-goog

